frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -628,7 +629,7 @@ export const ClusterServiceVersionsPage: React.FC<ClusterServiceVersionsPageProp
       Installed Operators are represented by Cluster Service Versions within this namespace. For
       more information, see the{' '}
       <ExternalLink
-        href="https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/architecture.md"
+        href={`${openshiftHelpBase}operators/olm-what-operators-are.html`}
         text="Operator Lifecycle Manager documentation"
```

Comment:
I'd suggest updating the link text as well.

```suggestion
        text="Understanding Operators documentation"
```